src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -57,6 +57,19 @@
         <!-- CoreExtension -->
         <service id="form.property_accessor" alias="property_accessor" public="false" />
 
+        <service id="form.choice_list_factory.default" class="Symfony\Component\Form\ChoiceList\Factory\DefaultChoiceListFactory" public="false"/>
+
+        <service id="form.choice_list_factory.property_access" class="Symfony\Component\Form\ChoiceList\Factory\PropertyAccessDecorator" public="false">
+            <argument type="service" id="form.choice_list_factory.default"/>
+            <argument type="service" id="property_accessor"/>
```

Comment:
`form.property_accessor` is an alias for `property_accessor` to be used for form related property access stuff: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml#L57-L63